src/common/robot.rs

```diff
@@ -1058,6 +1058,7 @@ mod tests {
                     },
                 )]),
             }),
+            ..Default::default()
```

Comment:
This syntax is new to me, could you explain?

Reply:
this will set all fields that are not explicitly set to their default value